lab-stuart/__test__/queue.test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const getQueue = require('../model/queue.js');
+
+describe('queue', () => {
+  describe('#enqueue, #dequeue, #count methods', () => {
+    test('the queue should have a first in first out behavior', () => {

```

Comment:
Try to make your tests a little bit more modular. Divide this one into 2-3 smaller ones. Each test should aim to examine very specific behavior.

That way, if a test fails, you know exactly what happened.